package.json

```diff
@@ -135,6 +135,20 @@
                     ],
                     "description": "Log Level"
                 },
+                "vscode-neovim.defaultMode": {
+                    "type": "string",
+                    "default": "normal",
+                    "enum": [
+                        "insert",
+                        "normal"
+                    ],
+                    "description": "What mode to start the editor in."
+                },
+                "vscode-neovim.resetModeOnWindowChange": {
+                    "type": "boolean",
+                    "default": true,
+                    "markdownDescription": "If 'true' will reset windows to the vscode-neovim.defaultMode on window change."
```

Comment:
Just for clarification, window change means when you leave Vscode (e.g. switch to browser)?

Reply:
It means switching to a different editor in vscode (perhaps editor change is a better name)

Reply:
Seems like Vscode terminology is in fact _editor_ (as indicated eg. by the command _Move Editor into Right Group_).

Reply:
Yeah, I might have been thinking about nvim's window, but I agree editor is better.